webapp/src/app/support/Files.ts

```diff
@@ -0,0 +1,27 @@
+import StreamBuffers from 'stream-buffers';
+
+export function read<R>(file: File, chunkSize: number, onChunk: (data) => {}, onFinish: () => R): R {
```

Comment:
This is a well-defined function which goal is read a File so this should go as a service on the SharedModule instead of a plain file. After that you can inject the service in classes whom need it.